events/0301_rb.html

```diff
@@ -0,0 +1,172 @@
+                <!-- <p class="box"><strong>Girl Develop It</strong> is here to empower and assist women interested in getting into and advancing in the tech industry through affordable and accessible programs.</p> -->
+                <p class="box"><strong>Girl Develop It</strong> is here to provide affordable programs for adult women interested in learning web and software development in a judgment-free environment.</p>
+
+                <div class="copy--small">
+                    <h4 class="green">Some "rules"</h4>
+                    <ul>
+                        <li>We are here for you!</li>
+                        <li>Every question is important.</li>
+                        <li>Help each other.</li>
+                        <li><strong>Have fun!</strong></li>
+                    </ul>
+                </div>
+
+                <p class="box copy--small"><small><strong>Girl Develop It is dedicated to providing a harrasment free learning experience for <em>everyone</em>.</strong><br/>For more information, see our <a href="http://www.girldevelopit.com/code-of-conduct" target="_blank">Code of Conduct</a>.</small></p>
+            </section>
+
+            <!-- Upcoming Events -->
+            <section>
+                <h3>Upcoming Events</h3>
+                <div class="box">
+                    <h3 class="event__title"><small><span class="green">Intro to</span></small><br/>Responsive Web Design<br/><small><span class="green">Workshop</span></small></h3>
+                    <h3 class="event__title"><small>3/10 @ Atomic Object</small></h3>
+                    <p><small>6pm - 9pm &nbsp; | &nbsp; 206 S. Fifth Ave #200</small></p>
+                    <aside class="notes"></aside>
+                </div>
+            </section>
+
+            <!-- Upcoming Events -->
+            <section>
+                <h3>Upcoming Events</h3>
+                <div class="box">
+                    <h3 class="event__title">Tech 101</h3>
+                    <h3 class="event__title"><small>3/19 @ Systems In Motion</small></h3>
+                    <p><small>6:30pm - 8:30pm &nbsp; | &nbsp; 1136 Oak Valley Dr.</small></p>
+                    <aside class="notes"></aside>
+                </div>
+            </section>
+
+            <!-- Upcoming Events -->
+            <section>
+                <h3>Upcoming Events</h3>
+                <div class="box">
+                    <h3 class="event__title"><span class="blue">Code &amp; Craft Beer</span></h3>
+                    <h3 class="event__title"><small>Tuesday 3/24 @ Corner Brewery</small></h3>
+                  <p><small>6:30pm - 8:30pm &nbsp; | &nbsp; 720 Norris, Ypsilanti</small></p>
+                    <aside class="notes"></aside>
+                </div>
+            </section>
+
+            <!-- Community Events -->
+            <section>
+                <h3>Community Events</h3>
+                <ul class="box--small copy--xsmall left" style="width: 48%">
```

Comment:
@krismyth lastly, let's clear out any events that take place on workshop days - 3/4 and 3/9; make sure the events are ordered by date and then i'll message you a few more events to add via Slack